Content.Server.Database/Model.cs

```diff
@@ -427,12 +530,13 @@ public class AdminLogEntity
     public interface IBanCommon<TUnban> where TUnban : IUnbanCommon
     {
         int Id { get; set; }
-        Guid? UserId { get; set; }
+        Guid? PlayerUserId { get; set; }
```

Comment:
These renamings are going to require a synchronized update of all the game servers since the first server to update will break running game servers. Just confirming that you're aware, here.

Reply:
I am very aware, but I think it's best if the name is consistent across all DB tables